src/vmprof_common.h

```diff
@@ -11,6 +11,8 @@ static int opened_profile(char *interp_name, int memory);
 static struct profbuf_s *volatile current_codes;
 #endif
 
+#define ENABLE_GZIP (defined(__unix__) || defined(__APPLE__))
```

Comment:
That's what I'm doing in this PR; the flag is only so that you can disable it if zlib is not available on your platform. But if we were to implement the gzip "binary" in Python then there would be no way to disable gzip. (Well, maybe it could be added later if it causes performance problems for someone.)